src/autopr/main.d

```diff
@@ -600,14 +600,26 @@ void processSubmodules ( Json edge, LibInfo lib_info, RequestLevel global,
             continue;
         }
 
+        bool d2_only = false;
+
+        if (!meta_info.neptune_yaml.isNull() &&
+            meta_info.neptune_yaml.containsKey("d2ready") &&
+            meta_info.neptune_yaml["d2ready"] == "only")
+        {
+            d2_only = true;
```

Comment:
This does not support `.D2-ready` anchor file, right? So we need to ensure all repos move that info to neptune yml.

Reply:
Correct, only the `.neptune.yml` content is considered